src/js/components/Calendar/stories/typescript/Simple.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import isChromatic from 'chromatic/isChromatic';
+
+import { Grommet, Box, Calendar } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const SimpleCalendar = () => {
+  const [date, setDate] = useState();
+
+  const onSelect = nextDate => {
+    setDate(nextDate !== date ? nextDate : undefined);
+  };
+
+  return (
+    <Grommet theme={grommet}>
+      <Box align="center" pad="large">
+        <Calendar
+          bounds={['2020-06-05', '2020-06-12']}
+          disabled={['2020-08-07']}
```

Comment:
Would this be an array of strings as opposed to a number?

Reply:
Good question! 😄 

In `Calendar` it is passing `daydisabled` to the disabled prop in `button` which is actually a number. 

https://github.com/grommet/grommet/blob/master/src/js/components/Calendar/Calendar.js#L390
so if you were to console what is being passed to disabled you will be able to see that there is a `number` as well as the `boolean`.  

This is why having a number in `button` for props should be sufficient. Then the `Calendar` `proptypes` will check if it is an array of strings

Reply:
Thanks for this explanation, I see how it all connects now. This line in the Calendar utils is helpful for an explanation of why a certain number is returned:

https://github.com/grommet/grommet/blob/master/src/js/components/Calendar/utils.js#L93